src/dist/RunLauncher.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=3.10
+Type=Application
+Terminal=true
+Exec=sh -c 'bash "$(dirname "%k")"/bin/TerasologyLauncher'
+Path=
+Icon=
+Name=Terasology Launcher
+Comment=Game Launcher
```

Comment:
i.e. "The launcher for the open-source voxel game Terasology."

No need to mention Linux since the user (hopefully) knows they're on Linux 😛